baselines/moon/moon/conf/cifar100_100clients.yaml

```diff
@@ -0,0 +1,33 @@
+---
+# this is the config that will be loaded as default by main.py
+# Please follow the provided structure (this will ensuring all baseline follow
+# a similar configuration structure and hence be easy to customise)
+
+num_clients: 100
+num_epochs: 10
+fraction_fit: 0.2
+batch_size: 64
+learning_rate: 0.01
+mu: 10
+temperature: 0.5
+alg: moon
+seed: 0
+server_device: cpu
+num_rounds: 500
+
+client_resources:
+  num_cpus: 8
+  num_gpus: 1
```

Comment:
Unlike for CIFAR-10, the `ResNet50` model here needs more VRAM (I'm seeing ~5GB). How about if we set the default to `num_gpus=0.5` instead? (so 2 flower clients share a GPU)